mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -606,19 +620,30 @@ do
                 if [ "\$POOL_STAKE_PREVIOUS_EPOCH" != "0" ] ; then
                     break
                 else
-                    echo ">>>> Not retrievable yet"
+                    echo ">>>> Stakes are not retrievable for this pool yet..."
                     sleep 2
                 fi
+                POOL_STAKE_RETRIEVAL_WAIT_ROUNDS=\$(( \$POOL_STAKE_RETRIEVAL_WAIT_ROUNDS + 1 ))
+                if [ "\$POOL_STAKE_RETRIEVAL_WAIT_ROUNDS" -gt "\$POOL_STAKE_RETRIEVAL_WAIT_ROUNDS_MAX" ] ; then
+                    echo ">>>> Error: too many attempts of stakes retrieval for this pool"
```

Comment:
Maybe something like "Timeout: could not retrieve stakes of pool '\$POOL_ID' within \$POOL_STAKE_RETRIEVAL_WAIT_ROUNDS_MAX attempts" could give a little more info ?